src/init/features/emulators.ts

```diff
@@ -100,6 +100,11 @@ export async function doSetup(setup: any, config: any) {
     ]);
   }
 
+  // Set the default behavior to be single project mode.
+  if (setup.config.emulators.singleProjectMode !== undefined) {
```

Comment:
If `singleProjectMode` in `firebase.json` is set to `false`, this condition means that `singleProjectMode` will be enabled. I'm sure that's not the intention.

Do you mean to just check if it's truthy?

```suggestion
  if (setup.config.emulators.singleProjectMode) {
```

Reply:
Fixed, nice find. After some testing this was backwards. The intention here is to set if it it doesn't exist, create the config, otherwise leave it as the user set it. The default should be true, but the current behavior is false.